Traceback (most recent call last):
  File "/home/priscyla/miniconda3/envs/analise_dados/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/priscyla/miniconda3/envs/analise_dados/lib/python3.11/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/priscyla/miniconda3/envs/analise_dados/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/priscyla/miniconda3/envs/analise_dados/lib/python3.11/asyncio/base_events.py", line 650, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/priscyla/miniconda3/envs/analise_dados/lib/python3.11/contextlib.py", line 222, in __aexit__
    await self.gen.athrow(typ, value, traceback)
  File "/home/priscyla/miniconda3/envs/analise_dados/lib/python3.11/site-packages/nbclient/client.py", line 650, in async_setup_kernel
    yield
  File "/home/priscyla/miniconda3/envs/analise_dados/lib/python3.11/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/priscyla/miniconda3/envs/analise_dados/lib/python3.11/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/priscyla/miniconda3/envs/analise_dados/lib/python3.11/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
datas = df_desenrola['DATA_BASE'].unique()

df_operacaoes = df_desenrola.groupby('DATA_BASE')['NUMERO_OPERACOES'].sum().reset_index()
df_operacaoes["PORCENTAGEM"] = ((df_op_faixa2["NUMERO_OPERACOES"] / total_numero) * 100).apply(lambda x: f"{x:.2f}%")
s = df_operacaoes[["NUMERO_OPERACOES"]].quantile(q=[0.25, 0.5, 0.75], axis=0, numeric_only=True, interpolation='linear')
print(s)

df_op_faixa1 = df_desenrola[df_desenrola['FAIXA_DESENROLA'] == 1].groupby('DATA_BASE')['NUMERO_OPERACOES'].sum().reindex(datas, fill_value=0).reset_index()
df_op_faixa1["PORCENTAGEM"] = ((df_op_faixa1["NUMERO_OPERACOES"] / df_op_faixa1['NUMERO_OPERACOES'].sum()) * 100).apply(lambda x: f"{x:.2f}%")

df_op_faixa2 = df_desenrola[df_desenrola['FAIXA_DESENROLA'] == 2].groupby('DATA_BASE')['NUMERO_OPERACOES'].sum().reindex(datas, fill_value=0).reset_index()
df_op_faixa2["PORCENTAGEM"] = ((df_op_faixa2["NUMERO_OPERACOES"] / df_op_faixa2['NUMERO_OPERACOES'].sum()) * 100).apply(lambda x: f"{x:.2f}%")

print('\n -- NÃšMERO DE OPERAÃ‡Ã•ES TOTAIS -- \n', df_operacaoes)
# print('\n -- NÃšMERO DE OPERAÃ‡Oáº¼S FAIXA 1 -- \n', df_op_faixa1)
# print('\n -- NÃšMERO DE OPERAÃ‡Oáº¼S FAIXA 2 -- \n', df_op_faixa2)
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[18], line 4[0m
[1;32m      1[0m datas [38;5;241m=[39m df_desenrola[[38;5;124m'[39m[38;5;124mDATA_BASE[39m[38;5;124m'[39m][38;5;241m.[39munique()
[1;32m      3[0m df_operacaoes [38;5;241m=[39m df_desenrola[38;5;241m.[39mgroupby([38;5;124m'[39m[38;5;124mDATA_BASE[39m[38;5;124m'[39m)[[38;5;124m'[39m[38;5;124mNUMERO_OPERACOES[39m[38;5;124m'[39m][38;5;241m.[39msum()[38;5;241m.[39mreset_index()
[0;32m----> 4[0m df_operacaoes[[38;5;124m"[39m[38;5;124mPORCENTAGEM[39m[38;5;124m"[39m] [38;5;241m=[39m (([43mdf_op_faixa2[49m[[38;5;124m"[39m[38;5;124mNUMERO_OPERACOES[39m[38;5;124m"[39m] [38;5;241m/[39m total_numero) [38;5;241m*[39m [38;5;241m100[39m)[38;5;241m.[39mapply([38;5;28;01mlambda[39;00m x: [38;5;124mf[39m[38;5;124m"[39m[38;5;132;01m{[39;00mx[38;5;132;01m:[39;00m[38;5;124m.2f[39m[38;5;132;01m}[39;00m[38;5;124m%[39m[38;5;124m"[39m)
[1;32m      5[0m s [38;5;241m=[39m df_operacaoes[[[38;5;124m"[39m[38;5;124mNUMERO_OPERACOES[39m[38;5;124m"[39m]][38;5;241m.[39mquantile(q[38;5;241m=[39m[[38;5;241m0.25[39m, [38;5;241m0.5[39m, [38;5;241m0.75[39m], axis[38;5;241m=[39m[38;5;241m0[39m, numeric_only[38;5;241m=[39m[38;5;28;01mTrue[39;00m, interpolation[38;5;241m=[39m[38;5;124m'[39m[38;5;124mlinear[39m[38;5;124m'[39m)
[1;32m      6[0m [38;5;28mprint[39m(s)

[0;31mNameError[0m: name 'df_op_faixa2' is not defined

